.github/workflows/workflow.yml

```diff
@@ -138,6 +138,12 @@ jobs:
         with:
           version: "0.12.14"
 
+      # Install CMake at a fixed version
+      - uses: jwlawson/actions-setup-cmake@v1.0
+        with:
+          cmake-version: "3.17.1"
+          github-api-token: ${{ secrets.GITHUB_TOKEN }}
+
```

Comment:
I think it might be better to use the mocked `utils::exec_cmd` in the tests, you will just need to add it to https://github.com/starship/starship/blob/master/src/utils.rs#L41.

Once you have added it to can remove this, this has the advantage of allowing the tests to be run on machines that don't have cmake installed.
```suggestion
```

Reply:
Fixed! :+1: